src/Tizen.NUI/src/public/VisualConstants.cs

```diff
@@ -298,7 +306,23 @@ public enum VisualFittingModeType
         /// <summary>
         /// The visual should be stretched to fill, not preserving aspect ratio.
         /// </summary>
-        Fill
+        Fill,
```

Comment:
Hello, 
This seems to be required ACR. (adding API)
because VisualFittingModeType is public API.
could you check again?